src/components/TPS6293x.stanza

```diff
@@ -366,3 +363,166 @@ public pcb-module device (
       net UVLO-EN (SW.EN, uvlo-div.out)
       net (GND, uvlo-div.p[2])
 
+
+defn select-inductor-type (L-val:Toleranced) -> [Instantiable, Double]:
+  val L-nH = to-nH(typ(L-val))
+  val series = inductors/components/NRS5040/NRS5040-specs
+  val [L-in-series, specs] = unpack $ get-ceil-specs(L-nH, series)
+  val L-type = inductors/components/NRS5040/component(L-in-series)
+  val L = to-H(L-in-series)
+  [L-type, L]
+
+defn compute-buck-solution (cst:BuckConstraints) -> [BuckSolution, Instantiable]:
+
+  val L-val = compute-L(cst)
+
+  ; Select from a known series
+  val [L-type, L] = select-inductor-type(L-val)
+
+  val I-peak = compute-peak-current(cst, L)
+
+  ; TODO - check I-peak here.
+  val I-margin = (30 %)
```

Comment:
Should the peak check use 1.30 as the multiplier or is using 0.30 ok?

Reply:
Good catch.